parent/pom.xml

```diff
@@ -1603,6 +1601,30 @@
             </plugins>
          </build>
       </profile>
+      <profile>
+         <id>jigsaw</id>
+         <activation>
+            <jdk>[1.9,)</jdk>
```

Comment:
are you sure this activates?

In my experience, I have to use "9" as format, not "1.9".